config/locales/en.yml

```diff
@@ -28,12 +28,13 @@ en:
     translations: "Translations"
   components:
     related_navigation:
-      collections: "Collections"
```

Comment:
Would it make sense to rename the key as well?

Reply:
Hmm, I tend to have gone with the plural for the data within the component, to indicate that the variable can contain multiple things, and therefore also the translation so that it's clear how they're linked. I think this makes sense, but if people find it clearer the other way then I can change it. Don't think it should be a blocker for now though.

Reply:
Agree not a blocker. It just feels like the key should match the English translation as far as possible and the plural/non plural looks a bit confusing to me.